torch/csrc/utils/python_arg_parser.cpp

```diff
@@ -503,6 +503,62 @@ static inline std::vector<int64_t> parse_intlist_args(const std::string& s, int6
   return args;
 }
 
+// Parse a string literal to remove quotes and escape sequences
+static std::string parse_string_literal(std::string str) {
```

Comment:
You shouldn't be taking `std::string` by value here as it induces a copy. Use `c10::string_view`

Reply:
Fixed